tests/nnfw_api/CMakeLists.txt

```diff
@@ -10,9 +10,10 @@ endif(NOT BUILD_ONERT)
 nnfw_find_package(GTest)
 
 set(RUNTIME_NNFW_API_TEST nnfw_api_gtest)
-file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_SRC "src/*.cc" "src/*.cpp")
+file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_LIB "lib/*.cc" "lib/*.cpp")
+file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_SRC "main.cc" "src/*.test.cc" "src/*.test.cpp")
```

Comment:
To reviews, I move main.cc from `src` to `.` because the main has deps to lib and src.